src/Geometry.h

```diff
@@ -27,6 +27,19 @@
 class Cmfd;
 
 /**
+ * @enum ringifyType
+ * @brief The method to use when subdividing cells into rings
+ */
+enum ringifyType{
```

Comment:
Per the [google c++ style guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names), there should be a space between the `ringifyType` and `{`.